keyboards/gray_studio/aero75/config.h

```diff
@@ -0,0 +1,196 @@
+
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0x4753
+#define PRODUCT_ID      0x3075
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Graystudio
+#define PRODUCT         Aero 75
```

Comment:
Please move to info.json.

Reply:
Can you specify this one?

Reply:
> Please move to info.json.

Can you specify this one? @drashna

Reply:
As documented in the [PR guide](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs) these defines need to live in info.json

Reply:
> As documented in the [PR guide](https://docs.qmk.fm/#/pr_checklist?id=keyboard-prs) these defines need to live in info.json

Thank you for clarifying! I have made the changes. Can you please take a look? @keyboard-magpie